frontend/taipy-gui/src/components/Taipy/AlertComponent.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { useCallback, useMemo } from "react";
+import Alert from "@mui/material/Alert";
+import { TaipyBaseProps } from "./utils";
+import { useClassNames } from "../../utils/hooks";
+
+interface AlertComponentProps extends TaipyBaseProps {
+    severity?: "error" | "warning" | "info" | "success";
+    message: string;
+    variant?: "filled" | "outlined" | "standard";
+}
+
+const AlertComponent = (props: AlertComponentProps) => {
```

Comment:
Please renane to Alert

Reply:
`Alert.tsx` already exists.

Reply:
then we need to rename that to Notification.tsx (and its dependencies)